### ISSUE TITLE
Accessibility Features - Increase font size

### ISSUE THREAD

**Issue by egasparini** [yalla-coop/cost-of-living-support#52]
**Summary**
Accessibility is very important especially for our target audience, therefore we want to make sure that our websites are easily readable and accessible to anyone. 

Accessibility Features are available from the same top bar as the language switcher. 
The topic of accessibility is split in multiple tasks: this one, about increasing the font size, and yalla-coop/cost-of-living-support#51 that covers more advanced features.

_Note: Accessibility features should be implemented both on UC & CoL._ 

**Notes**
This functionality is something we've created on other tools, so please use that. 

**Wireframes**
Desktop: https://www.figma.com/file/EqDZxsZf5o6cFfDbqyBsnt/Wireframes?node-id=5099%3A26651
Mobile: https://www.figma.com/file/EqDZxsZf5o6cFfDbqyBsnt/Wireframes?node-id=4675%3A135806

**Acceptance Criteria
Front End**
- [x] Website has an always visible top bar with accessibility features on the top left.
- [x] Accessibility features include: increase font size, a link to Accessibility that opens a separate page

Mobile: 
![Image](https://user-images.githubusercontent.com/55511970/199705165-825c789a-25a8-4a34-8821-0da750bfd072.png)

Desktop: 
![Image](https://user-images.githubusercontent.com/55511970/199705307-49acb779-12ce-4604-aa43-1c351e92bd0b.png)

- [x] Clicking once on the font size icon increases the font size by 25%
- [x] Clicking on it again brings the font size back to normal

**UI Criteria**
- [x] The tool tip is in under the Increase text size button 
<img width="510" alt="CleanShot 2022-10-26 at 11 35 04@2x" src="https://user-images.githubusercontent.com/25960351/198004899-821274cf-10bf-4a4b-8aaf-551742501cdc.png">

- [x] When you press Increase text size button it changes to Decrease text size found in these variants: https://www.figma.com/file/EqDZxsZf5o6cFfDbqyBsnt/Wireframes?node-id=4741%3A23980

**Back End**
- not needed, just store it in the local storage

**Comment by fadeomar** [comment]
hey @Israa91 , this issue is required in both UC app and COL app
